src/index.jsx

```diff
@@ -16,14 +16,7 @@ import Events from './components/events/Events';
 import Details from './components/details/Details';
 import Header from './components/Header';
 
-const { __REDUX_DEVTOOLS_EXTENSION__: REDUX_DEVTOOLS } = window;
-
-const composedEnhancers = compose(
-  applyMiddleware(thunk),
-  REDUX_DEVTOOLS && REDUX_DEVTOOLS()
-);
-
-const store = createStore(allReducers, composedEnhancers);
+const store = createStore(allReducers, applyMiddleware(thunk));
```

Comment:
youre not defaulting correctly thats why it didnt work...

in next pr, make it so that if dev tools extensions exist AND ITS NOT a production build use it...otherwise dont. Make it work like that, thats the proper expected way